about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link   href="about.css" rel="stylesheet">
+  <title>Home page</title>
+
+<body id="about"> 
+    
+    <header>
+      <div class="top">
+        <a class="facebook"><img src="./images/facebook.png"></a>
+        <a><img class  ="twitter" src="./images/twitter.png"></a>
+        <a class="Language"><p>English</p></a>
+        <a class="Page"><p>My Page</p></a>
+        <a class="checkout"><p>Logout</p></a>
+      </div>
+        <h1 class="background-img"></h1>
+        <div class="top-strip"></div> 
+       
+        <h1 class="logo"><img class="my_logo" src="./images/logo.png"> </h1>
+        <nav class="navbar">  
+            <div class="hamburger">
+                <span class="nav_bar"></span>
+                <span class="nav_bar"></span>
+                <span class="nav_bar"></span>   
+            </div>
+            <div class="top-bar">   
+        <ul class="nav_menu">
+            <li class="nav-link"><a href="index.html" class="nav_link">Home</a></li>
```

Comment:
- [ ] Please change this link to direct to `home.html` instead of the index.html because you don't have index.html file 😃 👍